back/src/database/makeSQL.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+const SearchWords = require("../objects/SearchWords");
+
+module.exports.makeSQLforSpot = function(table, keywords){
```

Comment:
This function is using a pattern similar to a builder pattern. Can you rename it to something more clear so that we know what the function is for without looking at the code? Something like getSpotQuery or getSpotQueryBuilder or getSpotWhere so that we know that this is a method to get data